messaging/src/main/java/org/axonframework/eventhandling/ReplayToken.java

```diff
@@ -99,6 +99,18 @@ public static boolean isReplay(TrackingToken trackingToken) {
 
     }
 
+    /**
+     * Return the relative position at which a reset was triggered whether this Segment is still replaying.
```

Comment:
I'd adjust this javadoc somewhat similar to the suggested adjustments in the `EventTrackerStatus` interface.